### PR TITLE
Increases test coverage

### PR DESCRIPTION
**Pull request by JinHao-L**
Add tests for:
- Archive & backup related methods
- NuudleParserTest
- Other small areas

### CODE REVIEW

File: src/main/java/seedu/address/storage/StorageStatsManager.java [hunk 1]
```diff
@@ -18,10 +18,10 @@ public String getArchiveStatusMessage() {
                     numArchivedAppointments > 1 ? "appointments" : "appointment");
         }
         if (numExpiredAppointments > 0) {
-            message += String.format(", of which %d %s not done.", numExpiredAppointments,
+            message += String.format(", of which %d %s not done", numExpiredAppointments,
                     numExpiredAppointments > 1 ? "are" : "is");
         }
-        return message;
+        return message.isBlank() ? message : message + ".";
```
Comment (WadeForward) [inline src/main/java/seedu/address/storage/StorageStatsManager.java:24]: Good deal with the punctuation 👍

File: src/test/java/seedu/address/commons/core/ConfigTest.java [hunk 2]
```diff
@@ -20,7 +24,31 @@ public void toString_defaultObject_stringReturned() {
...
+        // same object -> returns true
         assertTrue(defaultConfig.equals(defaultConfig));
+
+        // null -> returns false
+        assertFalse(defaultConfig.equals(null));
+
+        // different types -> returns false
+        assertFalse(defaultConfig.equals(5));
+
+        // different level -> returns false
```
Comment (xz0127) [inline src/test/java/seedu/address/commons/core/ConfigTest.java]: A really minor comment: would it be better to make the comment singular/plural form consistent?

**Review by xz0127 — APPROVED** [review]
Excellent work! Have a minor comment but feel free to ignore.
LGTM! 👍

File: src/test/java/seedu/address/commons/core/ConfigTest.java (hunk 2 above)
Comment (JinHao-L) [reply src/test/java/seedu/address/commons/core/ConfigTest.java]: Yep, I agree. Already updated it on my end.

**Review by WadeForward — APPROVED** [review]
This is definitely a solid commit,
LGTM!

**Pull request merged**